recipes/netpbm/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "10.68" %}
+
+package:
+  name: netpbm
+  version: {{ version }}
+
+source:
+  # See http://netpbm.svn.sourceforge.net/viewvc/netpbm/
+  svn_url: http://svn.code.sf.net/p/netpbm/code/advanced
+  svn_rev: 2294
+
+build:
+    number: 0
+    skip: true      # [win]
+
+requirements:
+  build:
+    - jpeg
+    - libtiff
+    - libpng
+    - zlib
+    - lxml
+    - svn
+  run:
+    - jpeg
+    - libtiff
+    - libpng
+    - zlib
+    - lxml
+test:
+  commands:
+    # There are a ton of commands, but just check one to make sure it works.
+    - pnmtopng --version
+
+about:
+  home: http://netpbm.sourceforge.net/
+  license: GPL
```

Comment:
I'm just not clear on what the non-free code is and how we remove it. Maybe we should be getting the source from someone different entirely like one of the Linux distros that has done their homework on this.